status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "f859b58c3896523b2fb9a09b3e5c1cedb04f959b",
-    "commit-sha1": "f859b58c3896523b2fb9a09b3e5c1cedb04f959b",
+    "version": "v1.0.0",
+    "commit-sha1": "efe2689df29586981ebcac398de473d6b7586bfd",
```

Comment:
good catch!

Reply:
actually its okay, the label is misleading but its not a commit sha, its the sha of the tag.
incase of tagged versions we use the sha of the tag and in case of branch names we use the sha of the latest commit.